setup.py

```diff
@@ -27,9 +27,9 @@
         [console_scripts]
         colin=colin.cli.colin:cli
     ''',
-    data_files=[("share/colin/config/", ["config/default.json",
-                                         "config/fedora.json",
-                                         "config/redhat.json"]),
+    data_files=[("share/colin/rulesets/", ["ruleset/default.json",
+                                           "ruleset/fedora.json",
+                                           "ruleset/redhat.json"]),
```

Comment:
Actually, wouldn't it be better if the directory in the repo is also called `rulesets` (not `ruleset`) ?

Reply:
Yes, I've changed that as well.